src/Typed/Resources/assets/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "@symfony/ux-typed",
+    "description": "Typed integration for Symfony",
+    "license": "MIT",
+    "version": "1.0.0",
+    "symfony": {
+        "controllers": {
+            "typed": {
+                "main": "dist/controller.js",
+                "webpackMode": "eager",
+                "fetch": "eager",
+                "enabled": true
+            }
+        }
+    },
+    "peerDependencies": {
+        "typed.js": "^2.0",
+        "stimulus": "^2.0.0"
+    },
+    "devDependencies": {
+        "typed.js": "^2.0",
+        "@symfony/stimulus-testing": "^1.1.0",
+        "stimulus": "^2.0.0"
```

Comment:
```suggestion
        "@hotwired/stimulus": "^3.0.0"
```

Reply:
I'm surprised the tests were passing... since your package would NOT have `@hotwired/stimulus` installed... but you ARE importing that from your controller... 🤔

Reply:
Actually in the test I imported 'stimulus' instead of '@hotwired/stimulus'